templates/santa/finish.html.twig

```diff
@@ -55,7 +55,7 @@
                 <h3>Explanation</h3>
 
                 <p>
-                    For information, here is the evil error{{ secretSanta.uniqueErrors|length > 1 ? 's' }} that caused this failure:
+                    For information, here {{ secretSanta.uniqueErrors|length > 1 ? 'are' : 'is' }} the evil error{{ secretSanta.uniqueErrors|length > 1 ? 's' }} that caused this failure:
```

Comment:
we should probably store errors in a temp variable to avoid calling array_unique 3 times

Reply:
Maybe make a method that applies array_unique to $errors before rendering the /finish/ page, that way we'd only call it once and we could loop on $errors on this line, what do you think ?

Reply:
I prefer to store a simple twig variable there, we don't need to manipulate those unique errors elsewhere